asterix/compound.ksy

```diff
@@ -0,0 +1,86 @@
+meta:
+  id: compound
+  license: GPL-3.0-only
+  endian: be
+  imports:
+    - field_spec
+    - fixed
+    - repetitive
+    - explicit
+    - extended
+
+  ks-version: 0.9
+
+doc: |
+  Implementantion of ASTERIX format.
+  
+  From Eurocontrol Asterix web page at
+  https://www.eurocontrol.int/asterix
+
+  Compound Standard Field format.
+
+  The compound field format is a variable lenght and content field.
+  This uses four parameters:
+
+  item, which is just the item name, as I001/010, or I062/390. It is just
+  informative, and is not used during parsing, but permits consult the item name
+  in the target language.
+
+  f_fmt, is a string with a list of the formats of the subfields,
+  The available formats are:
```

Comment:
Each field can have a `doc`. Probably this info should be moved there

Reply:
I preferred to explain a basic use form doc, imagined the field docs to have more specific information.

Reply:
Future versions would have additional explanations, for example an README with code examples is one of my objectives.